src/components/language/language.service.ts

```diff
@@ -361,6 +368,11 @@ export class LanguageService {
       session
     );
 
+    if (input.hasExternalFirstScripture) {
+      // Check if the language has no engagements that have firstScripture=true.
+      await this.checkLEFirstScripture(input.id);
```

Comment:
- Maybe this comment could be moved to the doc of the method?
- Maybe method `verifyExternalFirstScripture`? I'm not a huge fan of the language engagement abreviation.
- By moving this above the readOne it would save a DB call if it was going to fail.

Reply:
> The verification is needed on create as well.
> Also just some nitpicks.

@CarsonF @michaelmarshall I am not sure this is still needed on creating because the new language will not have any engagements and it will always pass the verification. 🤔

Reply:
That's an excellent point. It must be Friday lol